cmd/registry/registryList.go

```diff
@@ -122,6 +141,48 @@ func printNodes(nodes []*k3d.Node, headersOff bool) {
 	})
 
 	for _, node := range nodes {
-		fmt.Fprintf(tabwriter, "%s\t%s\t%s\n", strings.TrimPrefix(node.Name, "/"), string(node.Role), node.Labels[k3d.LabelClusterName])
+		// try top find the port in the host that leads to port 5000 in the container
+		registryPortBinding, ok := node.Ports[registryPort]
+		if !ok {
+			log.Fatalf("Registry %s does not have any port binding for port %s: %+v",
+				node.Name, registryPort, node.Ports)
+		}
+		hostRegistryPort := registryPortBinding[0].HostPort
+		hostRegistryPortI, err := strconv.Atoi(hostRegistryPort)
+		if err != nil {
+			log.Fatalf("Could not parse registry port %s: %s", hostRegistryPort, err)
+		}
```

Comment:
I'm actually not sure, if this is needed: If the registry was created by `k3d` (and we're only checking for those here, right?) then it will only have a single port mapped (and users cannot change the internally used port, so cannot be messed up by other unmapped but exposed ports).
Not a problem, that this code is here though, just saying :man_shrugging:

Reply:
Yeah, this is not needed at all, but it did not look right to not handle the error, even though it will not happen in 99% of cases...

Reply:
Fair enough :+1: